CLAUDE.md

```
@@ -53,13 +62,26 @@ sqlite3 modlog.db "DELETE FROM processed_actions WHERE created_at < date('now',
 
 The application supports both JSON config files and CLI arguments (CLI overrides JSON):
 
+### Core Options
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)